python_modules/dagster-graphql/dagster_graphql_tests/graphql/test_assets.py

```diff
@@ -1064,3 +1079,32 @@ def test_cross_repo_assets(self, graphql_context):
             upstream_asset["dependedByKeys"], key=lambda node: node.get("path")[0]
         )
         assert result_dependent_keys == dependent_asset_keys
+
+
+class TestNamedGroups(ExecutingGraphQLContextTestMatrix):
```

Comment:
We no longer need to have a new class here, we can put this text within this class located in the same file: https://github.com/dagster-io/dagster/blob/8fc3838f37099e5b48f6e363f949e449f1e908d3/python_modules/dagster-graphql/dagster_graphql_tests/graphql/test_assets.py#L321 because they both subclass the same context variants.